src/main/docker/RNApeg.sh

```diff
@@ -14,7 +14,7 @@ REFGENE=""
 OUTPUT_DIR=/results
 
 usage() {
-    echo "RNApeg.sh [-h] -b bamfile -f fasta -r refflat [-rg refflat]"
+    echo "RNApeg.sh [-h] -b bamfile -f fasta -r refflat [-rg refflat] -O output_dir"
```

Comment:
```suggestion
    echo "RNApeg.sh [-h] -b bamfile -f fasta -r refflat [-rg refflat] [-O output_dir]"
```

I'd keep this optional.